package.json

```diff
@@ -20,44 +20,42 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@types/react": "^16.9.34",
     "@zeit/next-sass": "^1.0.1",
     "classnames": "^2.2.6",
-    "deep-diff": "^1.0.2",
```

Comment:
are you sure you want to delete this?

Reply:
As far as I see,  it's not being used in the project. Did you see something wrong with it?

Reply:
Nono its just that i find it weird that someone added it. Maybe they found a usecase that we are not seeing. But you can remove it, its easy to put it back in